Demo.md

```diff
@@ -17,88 +17,81 @@ example, local networks.
 - `Central` is a deployment that receives messages from `nodes`, processing and
 presenting data related to them.
 
-We opted to deploy this poc as a StarlingX Application instead of deploying It directly through Kubernetes or with Helm Charts
-because our team had two major motivations:
+## How to deploy your workload
 
-1. We wanted to learn more about how to build a StarlingX Application.
-2. We want to make further improvements in the application to integrate
-   It with StarlingX sysinv to automatize specific nodes configurations
-   depending on the system states.
+We opted to deploy this proof of concept as a StarlingX Application. It's
+important to understand that any user workload can be deployed in many ways to
+the Kubernetes cluster(s) that StarlingX manages:
 
-But we would like to make It clear that if your application does not require a
```

Comment:
do you think this piece of information would not be useful?

Reply:
Not on the Demo, no. This information is already clear on the Readme of the App Gen.